src/Convert/SystemTasks.hs

```diff
@@ -0,0 +1,85 @@
+{- sv2v
+- Author: Ethan Sifferman <ethan@sifferman.dev>
+-
+- Conversion of SystemVerilog System Tasks to Verilog.
+-}
+
+module Convert.SystemTasks (convert) where
+
+import Convert.Traverse
+import Language.SystemVerilog.AST
+
+convert :: [AST] -> [AST]
+convert = map $ traverseDescriptions $ traverseModuleItems convertModuleItem
+
+convertModuleItem :: ModuleItem -> ModuleItem
+
+convertModuleItem (ElabTask SeverityInfo args) =
+    Initial (Block Seq "" [] [
+        (Subroutine (Ident "$write") (Args [(String "Elaboration Info: ")] [])),
+        (Subroutine (Ident "$display") args)
+    ])
+
+convertModuleItem (ElabTask SeverityWarning args) =
+    Initial (Block Seq "" [] [
+        (Subroutine (Ident "$write") (Args [(String "Elaboration Warning: ")] [])),
+        (Subroutine (Ident "$display") args)
+    ])
+
+convertModuleItem (ElabTask SeverityError args) =
+    Initial (Block Seq "" [] [
+        (Subroutine (Ident "$write") (Args [(String "Elaboration Error: ")] [])),
+        (Subroutine (Ident "$display") args)
+    ])
+
+convertModuleItem (ElabTask SeverityFatal (Args [] [])) =
+    Initial (Block Seq "" [] [
+        (Subroutine (Ident "$display") (Args [(String "Elaboration Fatal:")] [])),
+        (Subroutine (Ident "$finish") (Args [] []))
+    ])
+
+convertModuleItem (ElabTask SeverityFatal (Args (finishArgs:displayArgs) _)) =
+    Initial (Block Seq "" [] [
+        (Subroutine (Ident "$write") (Args [(String "Elaboration Fatal: ")] [])),
+        (Subroutine (Ident "$display") (Args displayArgs [])),
+        (Subroutine (Ident "$finish") (Args [finishArgs] []))
+    ])
+
+convertModuleItem other =
+    traverseStmts (traverseNestedStmts convertStmt) other
+
+timeCall :: Expr
+timeCall = Call (Ident "$time") (Args [] [])
+
+convertStmt :: Stmt -> Stmt
+convertStmt (Subroutine (Ident "$info") args) =
+    Block Seq "" [] [
+        (Subroutine (Ident "$write") (Args [(String "[%0t] Info: "), timeCall] [])),
+        (Subroutine (Ident "$display") args)
```

Comment:
Would it be possible to add these extra arguments to the beginning of `args` rather than outputting a block with two statements? E.g., the following works:
```SystemVerilog
initial $display("prefix: ", "%b %b", 1, 2);
```

Reply:
I didn't know you could do that! I will make the change